Apps/Playground/Android/app/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
 if (${ANDROID_JSENGINE_LIBNAME} STREQUAL "jsc")
     set(NAPI_JAVASCRIPT_ENGINE "JavaScriptCore" CACHE STRING "JavaScript engine for N-API.")
 elseif (${ANDROID_JSENGINE_LIBNAME} STREQUAL "v8android")
-    set(NAPI_JAVASCRIPT_ENGINE "V8" CACHE STRING "JavaScript engine for N-API.")
+    set(NAPI_JAVASCRIPT_ENGINE "v8" CACHE STRING "JavaScript engine for N-API.")
```

Comment:
Sorry, this isn't what I mean. We should continue to use V8 as the input, but convert it to lowercase when accessing the file.